github/repos_rules.go

```diff
@@ -992,42 +992,42 @@ func (s *RepositoriesService) UpdateRulesetClearBypassActor(ctx context.Context,
 
 // UpdateRulesetNoBypassActor updates a ruleset for the specified repository.
 //
-// This function is necessary as the UpdateRuleset function does not marshal ByPassActor if passed as nil or an empty array.
+// This function is necessary as the UpdateRuleset function does not marshal ByPassActor if passed as an empty array.
 //
 // Deprecated: Use UpdateRulesetClearBypassActor instead.
 //
 // GitHub API docs: https://docs.github.com/rest/repos/rules#update-a-repository-ruleset
 //
 //meta:operation PUT /repos/{owner}/{repo}/rulesets/{ruleset_id}
-func (s *RepositoriesService) UpdateRulesetNoBypassActor(ctx context.Context, owner, repo string, rulesetID int64, rs Ruleset) (*Ruleset, *Response, error) {
+func (s *RepositoriesService) UpdateRulesetNoBypassActor(ctx context.Context, owner, repo string, rulesetID int64, ruleSet Ruleset) (*Ruleset, *Response, error) {
```

Comment:
Sorry, one more nit:
Since GitHub doesn't capitalize "Set", let's please not either.
So: `ruleSet` => `ruleset` throughout.

Reply:
Sorry, I'll fix this.